features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -3216,6 +3216,7 @@
 
     <EnableTenantQualifiedUrls>{{tenant_context.enable_tenant_qualified_urls}}</EnableTenantQualifiedUrls>
     <EnableTenantedSessions>{{tenant_context.enable_tenanted_sessions | default(false)}}</EnableTenantedSessions>
+    <AppendSuperTenantInUrl>{{tenant_context.append_super_tenant_in_url | default(false)}}</AppendSuperTenantInUrl>
```

Comment:
I feel like "append" doesn't imply the condition strong enough. What about something like "require" or "mandate"?

Also, the config structure could be improved here.

tenant_context:
  tenant_qualified_urls:
        enable: {{tenant_context.tenant_qualified_urls.enable | tenant_context.enable_tenant_qualified_urls }} // to preserve backward compatibility
        require_super_tenant_domain


WDYT